rj_param_utils/include/rj_param_utils/param.h

```diff
@@ -180,13 +180,13 @@ class ParamProvider {
  * @param description A description of the parameter.
  */
 #define DEFINE_VARIABLE(type, module, name, val, description)              \
-    namespace params::variables {                                          \
+    namespace params::storage {                                            \
```

Comment:
Didn't this stuff get merged in in #1550?

Reply:
that's true, this change is accidental.

Reply:
actually idk why it shows this stuff as being changed. I don't see these changes when doing a `git diff ros2`